tests/command.bats

```diff
@@ -33,6 +33,34 @@ load "$BATS_PATH/load.bash"
   unstub docker
 }
 
+@test "can pass through optional params" {
+  export BUILDKITE_PLUGIN_JUNIT_ANNOTATE_ARTIFACTS="junits/*.xml"
+  export BUILDKITE_PLUGIN_JUNIT_ANNOTATE_JOB_UUID_FILE_PATTERN="custom_(*)_pattern.xml"
+  export BUILDKITE_PLUGIN_JUNIT_ANNOTATE_FAILURE_OUTPUT="file"
+
+  artifacts_tmp="tests/tmp/$PWD/junit-artifacts"
+  annotation_tmp="tests/tmp/$PWD/junit-annotation"
+
+  stub mktemp \
+    "-d junit-annotate-plugin-artifacts-tmp.XXXXXXXXXX : mkdir -p $artifacts_tmp; echo $artifacts_tmp" \
+    "-d junit-annotate-plugin-annotation-tmp.XXXXXXXXXX : mkdir -p $annotation_tmp; echo $annotation_tmp"
+
+  stub buildkite-agent "artifact download junits/*.xml /plugin/tests/tmp//plugin/junit-artifacts : echo Downloaded artifacts" \
+                       "annotate --context junit --style error : echo Annotation added"
+
+  stub docker "--log-level error run --rm --volume /plugin/tests/tmp//plugin/junit-artifacts:/junits --volume /plugin/hooks/../ruby:/src --env 'BUILDKITE_PLUGIN_JUNIT_ANNOTATE_JOB_UUID_FILE_PATTERN=${BUILDKITE_PLUGIN_JUNIT_ANNOTATE_JOB_UUID_FILE_PATTERN:-}' --env 'BUILDKITE_PLUGIN_JUNIT_ANNOTATE_FAILURE_OUTPUT=${BUILDKITE_PLUGIN_JUNIT_ANNOTATE_FAILURE_OUTPUT:-}' ruby:2.5-alpine /src/bin/annotate /junits : echo '<details>Failure</details>'"
```

Comment:
I tried to make the changes to pass the strings directly to the env vars in the stub docker command per your suggestions, but was not able to get the tests to pass :(

I tried the following syntax...
`--env MY_ENV_VAR=string`
`--env 'MY_ENV_VAR=string'`
`--env MY_ENV_VAR='string'`
... but no dice. Relevant commit here:  https://github.com/buildkite-plugins/junit-annotate-buildkite-plugin/pull/33/commits/f199e475bd1821813199a4311699b69cf0244418

In looking at what's going on in [hooks/command](https://github.com/buildkite-plugins/junit-annotate-buildkite-plugin/blob/master/hooks/command#L32) though... am I correct in understanding `--env "MY_ENV=${MY_ENV:-}"` is saying if MY_ENV is defined, use whatever that value is, otherwise, default to null. If that's the case, then would it be closer to mimicking set up to pass the env in the same way in test? I'm thinking this also adds the coverage in the existing tests that things do not blow up when the env var is not set at all.

Thanks a ton for all the thoughtful review and knowledge. Please lmk if this is alright or if I'm misunderstanding anything :)